distinguish mean and media by color

davearch opened the pull request:
Is this what you meant 'Distinguish between mean and median'?

This was the simplest change I could make to the code.

wholmgren commented:
I didn't have anything in particular in mind. Are you serving your fork somewhere?

davearch commented:
I set it up on a docker container, which runs on one of our virtual machines:
150.135.6.164

wholmgren commented:
What does docker have to do with it? 

See if you can make the color scale not suck i.e. make it more perceptually accurate. I would like to get that resolved before dedicating a new color to the median.

davearch commented:
I was just saying the fork runs in a docker container.

Do you mean you want more colors? Not sure how else to make it more 'perceptually accurate', except maybe making the lines thicker.

wholmgren commented:
Not necessarily more colors. See this [page](http://matplotlib.org/users/colormaps.html) and this [video](http://matplotlib.org/style_changes.html#colormap).

davearch commented:
What about now? http://150.135.6.164

I added a darker background, which makes the lighter colors easier to see. I also changed the color scheme to one that doesn't clash so much with a white/light or even a black/dark background. Not only that, but now we can use black and white for the mean/median, which I think makes the most sense.
